frontend/public/env.json

```diff
@@ -43,6 +43,11 @@
         "type": "string",
         "value": "DEV-IDIR"
     },
+    "fam_console_idp_name_bceid": {
+        "sensitive": false,
+        "type": "string",
+        "value": "TEST-BCEIDBUSINESS"
```

Comment:
use TEST-BCEIDBUSINESS for local, so we can test the bceid login, we don't have any dev business bceid